crowbar_framework/Gemfile

```diff
@@ -45,6 +45,7 @@ gem "rails-observers", "<= 0.1.2"
 gem "ohai", "~> 6.24.2"
 gem "chef", "~> 10.32.2"
 gem "net-http-digest_auth", "~> 1.4"
+gem "sexp_processor", "<= 4.12.0"
```

Comment:
This is only needed by brakman, right? Please move it down there, so it is only installed under the same condition. Also please add a comment, to remove it, once a newer brakman is in use.

Reply:
currently only brakeman is failing, So I hope its only brakeman. 
Moved and added comment.